src/primo/browzine-primo-adapter.js

```diff
@@ -448,6 +461,17 @@ browzine.primo = (function() {
     return featureEnabled;
   };
 
+  function showExpressionOfConcern() {
+    var featureEnabled = false;
+    var config = browzine.articleExpressionOfConcernEnabled;
```

Comment:
Now that we have a new configuration option, you'll want to check with either JohnS or Nick about where they want to add it, if anywhere.

At a minimum, https://thirdiron.atlassian.net/wiki/spaces/BrowZineAPIDocs/pages/79200260/Ex+Libris+Primo+Integration and https://thirdiron.atlassian.net/wiki/spaces/BrowZineAPIDocs/pages/104988678/Ex+Libris+Summon+Overview+and+Step-by-Step+Installation+Guide in the various Options (look for the expandable sections, they have the configs thoroughly specified there)